Frameworks/include/CGIWICBitmap.h

```diff
@@ -126,22 +160,52 @@ class CGIWICBitmap
             WICRect rect = { 0, 0, m_width, m_height };
             region = &rect;
         } else {
+            // This is for tracking purpose, we do not support sub-regions.
+            // TODO #1379 - should support regional locking.
+            // This check will be removed.
+            if (region->X || region->Y || (region->Height < m_height) || (region->Width < m_width)) {
+                return E_NOTIMPL;
+            }
+
             RETURN_HR_IF(E_INVALIDARG, (region->Width > m_width) || (region->Height > m_height));
```

Comment:
>   RETURN_HR_IF(E_INVALIDARG, (region->Width > m_width) || (region->Height > m_height)); [](start = 9, length = 88)

this might be doing something similar to above, but above code will eventually go away.